CMake/Macros.cmake

```diff
@@ -27,7 +27,7 @@
 # This macro is intended to be called multiple times with a sequence of
 # possibly dependent header files.  Some headers depend on others to be
 # compiled correctly.
-macro(check_include_file_concat _file _variable)
+macro(check_include_file_concat_curl _file _variable)
```

Comment:
Can you change it to curl prefix instead suffix to align with the other names, and be more clear that is curl macro?

Reply:
It's a local variation of a built-in function. A prefix made this much less obvious and reduced readability. Hence the suffix for this particular name.